onetl/db/db_reader/db_reader.py

```diff
@@ -501,6 +502,90 @@ def validate_options(cls, options, values):
 
         return None
 
+    @slot
+    def has_data(self) -> bool:
+        """Returns ``True`` if there is some data in the source, ``False`` otherwise.
```

Comment:
```suggestion
        """Returns ``True`` if there is some data in the source, ``False`` otherwise. |support_hooks|
```

И для `.raise_if_no_data()` тоже